<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#98FF98;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="homepage">
    <widget class="QPushButton" name="debug_page_bt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #6F4E37;
    border-radius: 20px;
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    color: white;
}
QPushButton:hover {
    background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>491</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:red;
color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Gym Nation Employee Attendance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:red;
color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>                  Latest Login By Employee</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_logout_bt">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #6F4E37;
    border-radius: 20px;
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    color: white;
}
QPushButton:hover {
    background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Login/logout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="salary_bt">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #6F4E37;
    border-radius: 20px;
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    color: white;
}
QPushButton:hover {
    background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Update Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_employee">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #6F4E37;
    border-radius: 20px;
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    color: white;
}
QPushButton:hover {
    background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="attendance_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:35pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 italic 14pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Number of employee Attend Today</string>
     </property>
    </widget>
    <widget class="QPushButton" name="admin_report_bt_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>120</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #6F4E37;
    border-radius: 20px;
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    color: white;
}
QPushButton:hover {
    background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="employee_login_logout">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:red;
color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>             Employee Log-Out or Log-In</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Enter Employee ID:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_bt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: red;
    font: 16pt &quot;MS Shell Dlg 2&quot;;
    color: white;
}

QPushButton:hover {
    background-color: darkred;
}
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_bt_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: red;
    font: 16pt &quot;MS Shell Dlg 2&quot;;
    color: white;
}

QPushButton:hover {
    background-color: darkred;
}
</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logout_bt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: red;
    font: 16pt &quot;MS Shell Dlg 2&quot;;
    color: white;
}

QPushButton:hover {
    background-color: darkred;
}
</string>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_logout_input">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>149</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_logout_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="reg_page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:red;
color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Employee Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Enter Employee ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employee_id_input">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employee_L_name_input">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Employee Last Name:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="employee_register_submit_bt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>580</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: red;
    font: 16pt &quot;MS Shell Dlg 2&quot;;
    color: white;
}

QPushButton:hover {
    background-color: darkred;
}
</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_register_bt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>640</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: red;
    font: 16pt &quot;MS Shell Dlg 2&quot;;
    color: white;
}

QPushButton:hover {
    background-color: darkred;
}
</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Employee Rate:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employee_rate_input">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Employee Position</string>
     </property>
    </widget>
    <widget class="QComboBox" name="employee_position_input">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Janitor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Senior Dev</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>System Admin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Human Resourses</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="register_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Employee First Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employee_F_name_input">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="debug_page_1">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:red;
color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>                              Debug Page</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_attendance_bt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #6F4E37;
    border-radius: 20px;
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    color: white;
}
QPushButton:hover {
    background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Clear Attendance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:red;
color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>   This will reset the attendance </string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_bt_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>640</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: red;
    font: 16pt &quot;MS Shell Dlg 2&quot;;
    color: white;
}

QPushButton:hover {
    background-color: darkred;
}
</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:red;
color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>   This will update Attendance</string>
     </property>
    </widget>
    <widget class="QPushButton" name="check_attenance_bt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #6F4E37;
    border-radius: 20px;
    font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    color: white;
}
QPushButton:hover {
    background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
